hadoop-plugin-test/src/main/gradle/negative/missingFields.gradle

```diff
@@ -99,6 +99,10 @@ hadoop {
       depends 'job16'
     }
 
-    targets 'job17'
+    pinotBuildAndPushJob('job20') {
```

Comment:
There seem to be  job 18 and 19 defined in jobs1.gradle.
   sqlJob('job18') {
      jdbcDriverClass 'com.teradata.jdbc.TeraDriver' //Required
      jdbcUrl '/job/data/src'            //Required
      jdbcUserId 'foo'                   //Required
      jdbcEncryptedCredential 'eyJ2YWwiOiJiQzVCU09HbDVwYndxNFRXV00yZ'  //Required
      jdbcCryptoKeyPath '/hdfs/path/to/cryptokey/file'                 //Required
      set properties: ['user.to.proxy': 'testUser',
                       'jdbc.sql.1': 'DELETE test_table_publish ALL;',
                       'jdbc.sql.2': 'INSERT INTO test_table_publish SELECT * FROM test_table;']
      depends 'job17'
    }

    hdfsToTeradataJob('job19') {
      hostName 'dw.teradata.com'  // Required
      userId 'foo' //Required
      cryptoKeyFilePath '/hdfs/path/to/cryptokey/file'
      encryptedCredential 'eyJ2YWwiOiJiQzVCU09HbDVwYndxNFRXV00yZ'
      targetTable 'dwh_stg.scott_users'
      sourceHiveDatabase 'hive_database'
      sourceHiveTable 'hive_table'
      set properties: ["user.to.proxy"  : "testUser",
                       "hdfs.fileformat": "orcfile"]
      depends 'job18'
    }